config/config.go

```diff
@@ -308,6 +308,13 @@ func DefaultConfig() *Config {
 			Height: 0,
 			Hash:   types.Hash{},
 		},
+		Erc20RWSigner: Erc20RewardSignerConfig{
+			Enable:      false,
+			PrivateKeys: nil,
+			Targets:     nil,
+			// the reasonable value is the block time
+			SyncEvery: types.Duration(1 * time.Minute),
```

Comment:
Technically we don't need this configuration, but it maybe not what user wants.
Since rn the block time is determined by two consensus parameters(proposeTimeout and emptyBlockTimeout), not easy for the signerSvc to generate a good value based on that two parameters for this configuration. So I'll leave to the user.